jobs/build/signed-compose/build.groovy

```diff
@@ -274,15 +229,15 @@ ${tagResult.combined}
 """)
         }
 
-        echo("Building RHEL8 puddle")
-        def puddleCmd = "ssh ocp-build@rcm-guest.app.eng.bos.redhat.com sh -s ${buildlib.args_to_string(params.BUILD_VERSION)} < ${env.WORKSPACE}/build-scripts/rcm-guest/call_puddle_advisory_el8.sh"
+        echo("Building RHEL${elMajor} puddle")
+        def puddleCmd = "ssh ocp-build@rcm-guest.app.eng.bos.redhat.com sh -s ${buildlib.args_to_string(params.BUILD_VERSION)} el${elMajor} < ${env.WORKSPACE}/build-scripts/rcm-guest/call_puddle_signed_tag.sh"
```

Comment:
I think currently we can use `buildlib.invoke_on_rcm_guest()` to run commands on rcm-guest?

Reply:
Normally; _but_ that only returns output, not the return status. We use the `returnStatus` below.